h2o-genmodel/src/main/java/hex/genmodel/utils/LinkFunctionType.java

```diff
@@ -12,5 +12,6 @@
     ologit,
     ologlog,
     oprobit,
-    inverse
+    inverse,
+    tweedie
```

Comment:
same as the distribution, why it is needed now?

Reply:
For GLM, the link function for tweedie is also named Tweedie.  Please don't ask me why.  Tomas did it.

Reply:
@wendy, do you really use this enum in GLM? GLM has its own implementation of Distribution and LinkFunction.